app/controllers/errors_controller.rb

```diff
@@ -16,6 +16,11 @@ def error_content
         title: "Page not found",
         message: "The page you were looking for could not be found."
       }
+    when 406
```

Comment:
Rails generates html for the 406 status error page so I deleted that and added support for 406s into our existing error handling.